proto/api/v2/compliance_results_service.proto

```diff
@@ -105,6 +111,11 @@ message ListComplianceScanResultsResponse {
     repeated ComplianceScanResult scan_results = 1;
 }
 
+// ListComplianceCheckScanStatsResponse provides stats based on the check names
```

Comment:
provides overall states per cluster? Where does check names come into play here?

ComplianceCheckStatusCount -> maps cluster to list of ComplianceCheckStatusCount

ComplianceCheckStatusCount -> count per status (PASS, FAIL). I don't see a check name

Reply:
Check name doesn't come into play here at all.  I copied the comment from another WIP PR.